### ISSUE TITLE
[Android] Block all fingerprinting by default

### ISSUE THREAD

**Issue by newuser-web** [brave/brave-browser#11070]
## Description 
By default brave is blocking cross site fingerprinting. Change default to block all fingerprinting for all webpage.

## Steps to reproduce 

   1. 
   2. 
   3. 

## Issue happens on 
   - Current Play Store version?
   - Beta build?

## Device details
   - Install type (ARM, x86):
   - Device (Phone, Tablet, Phablet):
   - Android version:
   
### Website problems only
- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome?

**Comment by srirambv** [comment]
It is by design to only block 3rd party fingerprint. With 1.11.x user has granular controls similar to desktop so the user can change it as they will. Closing as wontfix.

cc: @SergeyZhukovsky @anthonypkeane

**Comment by newuser-web** [comment]
Then take it as feature request

**Comment by newuser-web** [comment]
Take it as feature request. I'm not able to change the setting for all webpage in a click.

**Comment by srirambv** [comment]
Global settings similar to desktop will be available once #6831 is done.
